src/edu/csus/ecs/pc2/ui/MultipleDataSetPane.java

```diff
@@ -751,4 +788,26 @@ public void actionPerformed(ActionEvent e) {
         }
         return chckbxStopOnFirstFailedTestCase;
     }
+
+    private JCheckBox getLoadSamplesFirstCheckbox() {
+        if (loadSamplesFirstCheckbox == null) {
+            loadSamplesFirstCheckbox = new JCheckBox("Load Sample Data");
+            loadSamplesFirstCheckbox.setToolTipText("Load Sample datasets before other data file");
```

Comment:
"file" in the above message should be plural ("...before loading other data files").